src-tauri/src/main.rs

```diff
@@ -62,14 +60,36 @@ struct PreviewLoaded {
     created_time: Option<SystemTime>
 }
 
-async fn get_preview(path: &std::path::PathBuf, manager: AppHandle) -> Result<()> {
+fn init_statics_and_dir() {
+    let cli = Cli::parse();
+    let working_dir = PathBuf::from(&cli.path);
+    ARK_SHELF_WORKING_DIR.set(working_dir).unwrap();
+    let scores_path = PathBuf::from(&cli.path).join(arklib::STORAGES_FOLDER).join("shelf").join("scores");
```

Comment:
Just `.join(arklib::STORAGES_FOLDER).join("scores");` without `.join("shelf")`. Storages should be usable across the apps.